ontology/vt.md

```diff
@@ -38,4 +38,5 @@ usages:
         description: "Studies in the MPD database that have measurements related to the VTO term spleen size trait or its descendants"
 repository: https://github.com/AnimalGenome/vertebrate-trait-ontology
 preferredPrefix: VT
+domain: phenotype
```

Comment:
@caripark thoughts?

Reply:
It's not ideal, but it's probably the closest option.

Reply:
I think its fine.. the domain should really be called `trait and phenotype` -  this is what it is called in my head.